dask/array/core.py

```diff
@@ -4141,7 +4148,7 @@ def retrieve_from_ooc(keys, dsk_pre, dsk_post=None):
     return load_dsk
 
 
-def asarray(a, allow_unknown_chunksizes=False, **kwargs):
+def asarray(a, allow_unknown_chunksizes=False, *, like=None, **kwargs):
```

Comment:
Let's add an entry for `like` to this docstring

Reply:
Thanks for the reminder, done in https://github.com/dask/dask/pull/8054/commits/73ea480387e515cf7dcd1ea814553408b79a0fb5 .